source/scripts/model/Monster.js

```diff
@@ -100,6 +100,28 @@ export default class Monster {
         this.position.x += movement.x
         this.position.y += movement.y
     }
+    pursuit() {
+          //Variables for the monster and hero positions
+       var monX = this.position.x
+       var monY = this.position.y
+       var adX = this.game.adventurer.position.x 
+       var adY = this.game.adventurer.position.y
```

Comment:
I still really really dislike this alias-ing here; just type `this.position.x`, so we don't have to think about it.